tests/test_genericls.py

```diff
@@ -82,8 +82,8 @@ def test_ground_to_image(self, hrsc_nadir_model, image, ground):
         y, x = hrsc_nadir_model.groundToImage(*ground)
         iy, ix, _ = image
 
-        assert x == pytest.approx(ix, abs=0.5)
-        assert y == pytest.approx(iy, abs=0.5)
+        assert x == pytest.approx(ix, abs=1)
+        assert y == pytest.approx(iy, abs=1)
```

Comment:
These tests had to be loosened up because the ground to image precision is now much higher and the actual results are different.

Reply:
We should change these from comparing against ISIS to comparing against CSM.